### ISSUE TITLE
Resize unavailable on touch devices (hybrid laptops)

### ISSUE THREAD

**Issue by studioscully** [tinymce/tinymce#6174]
**What is the current behaviour?**
Similar to issue #5310, on a Windows 10 device running in "tablet mode", TinyMCE considers the device mobile and disables image / editor resizing functionality.

**Steps to Reproduce:**
1) On a Windows 10 hybrid device (e.g. MS Surface or Dell XPS 2-in-1), set the device to Tablet Mode.
2) Load a page with TinyMCE embedded
3) TinyMCE editor / image resizing is not available

Switching back out of tablet mode and refreshing shows the resizing functionality again.

This occurs with mobile theme either not specified (default) or to "silver".

**What is the expected behavior?**
Resizing functionality is available, as the device is a hybrid desktop, with mouse / keyboard attached to it.

**Which versions of TinyMCE, and which browser / OS are affected by this issue?**
TinyMCE 5.5.1
Windows 10 1909/2004

**Did this work in previous versions of TinyMCE?**
Unsure (our users only just moved to XPS 2-in-1 devices, sorry).

**Comment by lnewson** [comment]
This is expected behaviour, as if you switch to tablet mode then the browser will report it's a mobile device and as such the mobile defaults will be applied: https://www.tiny.cloud/docs/mobile/#mobiledefaultsforselectedsettings. You can override the settings if you don't want this to happen by adding something like this to your init configuration:
```
mobile: {
  resize: true,
  object_resizing: true
}
```

**Comment by studioscully** [comment]
Ah great, thank you! That works a treat.

It was the documentation that confused me in the first place, as in the next section of that page (https://www.tiny.cloud/docs/mobile/#unsupportedsettingsformobile) it says that "resize" and "object_resizing" aren't supported settings on mobile devices.

Not sure if that needs updating, or if I'm just having a bad brain day :D

**Comment by lnewson** [comment]
That is technically true, as the functionality doesn't work with touch devices and that's why it gets disabled by default. I'll talk to our docs team though, as that should probably clarify it's not supported on touch devices rather than mobile devices. Thanks for the feedback.

**Comment by studioscully** [comment]
Hi again,

We've done a bit more testing and it looks like the solution provided (adding the overrides in the init config) doesn't work - even with those options added, the resize functionality is not available to us when using our PCs in "tablet" position.

This is an issue for us because:
- We're using convertable Windows laptops (XPS 15 2-in-1)
- The laptops are in "tablet" position, i.e the keyboard wrapped around the back, with external keyboard and mouse
- Windows then reports to the web browser that the PC is in "tablet" mode
- TinyMCE considers this a mobile device, hiding the resize functionality

The issue for us appears to be that the default behaviour of "hide the resize functionality on mobile devices" can't be overriden with those switches.

**Comment by shanereichart** [comment]
This affects Windows PCs which are "tablets" at the OS level, whether or not "Tablet mode" is enabled. On v4.x I was able to resize images and media, but on v5.x am no longer able. "Tablet mode" is off and, in fact, disabled because I'm employing multiple monitors.

Is this also what causes the context menu to be horizontal? It's resultant from a different class applied to it than my other device has. (It's not a tablet.)

![image](https://user-images.githubusercontent.com/16807422/102172476-d8cf9080-3e66-11eb-9c33-783f86a65273.png)

**Comment by shanereichart** [comment]
@lnewson, please note these issues haven't been completely addressed, so this issue appears to have been closed prematurely. Both issues are that TinyMCE renders the _mobile_ experience on touchscreen Windows 10 devices which are in desktop mode with a keyboard and mouse ("Tablet mode" is disabled).

1. **Image resize handles are disabled.** This can be overridden by explicitly forcing `object_resizing: true`, but this also overrides the experience in tablet mode, revealing the resize handles always.
2. **The context menu appears horizontally.** I'm aware of no way to override this behavior.

**Comment by studioscully** [comment]
I've been looking into this further and I can confirm @shanereichart 's findings re: resizing.

The following appears true from my testing:

**Default settings:**
1. Win10 devices - standard position: resizing of editor and objects **enabled**
2. Win10 devices - tablet position*: resizing of editor and objects **disabled**
3. Mobile devices: resizing of editor and objects **disabled**

**Enabling resizing:**
1. Win10 devices - standard position: not required
2. Win10 devices - tablet position*: use Global Overrides below
3. Mobile devices: use Mobile Overrides below

* This applies regardless of whether "Tablet Mode" is enabled.
 
**Global Overrides**
```
{
  object_resizing: true,
  resizing: true
}
```

**Mobile Overrides**
```
{
  mobile : {
    object_resizing: true,
    resizing: true
  }
}
```

**In summary:**
Win10 devices, when in tablet position, appear to default to having resizing disabled. To enable resizing in this case you use the standard "resize" and "object_resizing" parameters and **not** the mobile overrides.

As per @shanereichart's comments, Win10 devices with "Tablet Mode" enabled should probably be using the mobile overrides!

The current behaviour isn't clear in the documentation, so it would be useful to have it clarified there:
1. On Win10 devices in tablet position (regardless of "Tablet Mode"), resizing defaults to false
2. Win10 devices in tablet position are not considered mobile devices by TinyMCE, so the mobile settings do not apply (note: I might be wrong for non-sizing parameters, that'd need checking!)
